### PR TITLE
v83.0.0

### PR DESCRIPTION
**Pull request by jverkoey**

### CODE REVIEW

**Comment by jverkoey** [comment]
@bernaferrari

> I have no idea what is this:
https://github.com/material-components/material-components-ios/blob/develop/docs/theming.md#how-to-get-the-code-1

Fair point, those docs look pretty crufty. They're part of our unsupported Themer APIs documentation though so it's unlikely that we'll devote time toward cleaning it up.

**Comment by bernaferrari** [comment]
No problem. Examples is not in the same section as the deprecated themer API. That's what created confusion for me. I thought they were examples for the supported theme, because, well, different section.

**Comment by jverkoey** [comment]
Ah; yep more headers to be fixed :) If you're interested in sending us PRs that would be an easy one for you to start with!

**Comment by bernaferrari** [comment]
Thanks, #7240.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,931 @@
+# 83.0.0
+
+This major release graduates several Theming extensions from Beta and includes a breaking change for
```
Comment (codeman7) [inline CHANGELOG.md:3]: _Tip:_ "Theming Extensions" or "theming extensions"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,931 @@
...
+AppBar's imports. All Themer APIs have been annotated as "to be deprecated" in lieu of Theming
+extensions.
```
Comment (codeman7) [inline CHANGELOG.md:5]: How do you feel about listing all the components that are graduating their theming extension?

**Review by codeman7 — APPROVED** [review]

**Comment by codeman7** [comment]
Should we pull in #7241?

File: CHANGELOG.md (hunk 2 above)
Comment (jverkoey) [reply CHANGELOG.md:5]: Done.

**Comment by romoore** [comment]
@jverkoey Is this actually ready for review? I see active work being done on the PR.

**Comment by jverkoey** [comment]
@romoore Yes it is actively being worked on in response to review feedback.

**Review by codeman7 — APPROVED** [review]

**Pull request merged**